src/test/java/com/auth0/RequestProcessorTest.java

```diff
@@ -68,7 +74,7 @@ public void shouldThrowOnProcessIfRequestHasError() throws Exception {
         HttpServletRequest req = getRequest(params);
 
         RequestProcessor handler = new RequestProcessor(client, "code", verifyOptions);
-        handler.process(req);
+        handler.process(req, response);
```

Comment:
nit, rename `req` to `request`

Reply:
Renamed all `req` variables to `request` in this class.